mvpa2/algorithms/searchlight_hyperalignment.py

```diff
@@ -351,6 +367,10 @@ def _proc_block(self, block, datasets, measure, queryengines, seed=None, iblock=
 
             # Find the neighborhood for that selected nearest node
             roi_feature_ids_all = [qe[node_id] for qe in queryengines]
+            # handling queryengines that return AttrDatasets
+            for isub in range(len(roi_feature_ids_all)):
+                if is_datasetlike(roi_feature_ids_all[isub]):
```

Comment:
`assert roi_feature_ids_all[isub].nsamples == 1` right?

btw, why we need to keep indices in a list (saw below .tolist) -- keeping in ndarray would be more efficient... or is that something we expect somehwere???